stt.c

```diff
@@ -121,7 +125,7 @@ writetimes(FILE * fp, struct timesnode * p)
 	if (p->left != NULL) {
 		writetimes(fp, p->left);
 	}
-	fprintf(fp, "%lld;%lld;%s\n", p->starttime, p->endtime, p->task);
+	fprintf(fp, "%ld;%ld;%s\n", p->starttime, p->endtime, p->task);
```

Comment:
clang on openbsd give me this now:

```
$ make
stt build options:
CFLAGS   = -std=c99 -pedantic -Wall -Os  -DVERSION="0.1"
LDFLAGS  = -static -s 
CC       = cc
CC stt.c
stt.c:128:30: warning: format specifies type 'long' but the argument has type 'time_t' (aka 'long long') [-Wformat]
        fprintf(fp, "%ld;%ld;%s\n", p->starttime, p->endtime, p->task);
                     ~~~            ^~~~~~~~~~~~
                     %lld
stt.c:128:44: warning: format specifies type 'long' but the argument has type 'time_t' (aka 'long long') [-Wformat]
        fprintf(fp, "%ld;%ld;%s\n", p->starttime, p->endtime, p->task);
                         ~~~                      ^~~~~~~~~~
                         %lld
2 warnings generated.
CC -o stt
```

But it works as intended on both linux and obsd